package.json

```diff
@@ -14,11 +14,11 @@
     "lint": "npm run eslint && npm run stylelint",
     "start": "webpack-dev-server",
     "clean": "rm -rf dist/",
-    "build": "npm run clean && webpack --mode production --env prod",
+    "build:prod": "npm run clean && webpack --mode production --env prod",
     "build:staging": "npm run clean && webpack --mode production --env staging",
-    "serve": "npm run build && firebase serve",
+    "serve:prod": "npm run build && firebase serve",
     "serve:staging": "npm run build:staging && firebase serve",
-    "deploy": "npm run build && firebase deploy -P prod",
+    "deploy:prod": "npm run build && firebase deploy -P prod",
```

Comment:
Det samme gjelder vel her

Reply:
Ja, egentlig er det cleanere å bare holde dem som de var

Reply:
For å vere konsekvent synst eg det er greit å kalle det `build:prod` også.